imagebuildah/stage_executor.go

```diff
@@ -1556,6 +1568,32 @@ func (s *StageExecutor) commit(ctx context.Context, createdBy string, emptyLayer
 		Manifest:              s.executor.manifest,
 		UnsetEnvs:             s.executor.unsetEnvs,
 	}
+	// generate build output
+	if s.executor.buildOutput != "" {
+		extractRootfsOpts := buildah.ExtractRootfsOptions{}
+		if unshare.IsRootless() {
+			// Strip setuid,setgid,sticky and extended attributes
```

Comment:
Non-blocking? It’s kinda obvious that that’s what the three lines do, to me that doesn’t need a comment.

But I’d _love_ to see a comment recording the outcome of the discussions. What's the rationale for stripping these bits for rootless use, and not as root? (IOW, if someone came 5 years later thinking that this is surprising, what would be necessary to know?)

[OTOH if this is a hard-won consensus but asking for details would break that consensus :), by all means let’s leave it as it is.]